lang/attribute/program/src/declare_program/common.rs

```diff
@@ -168,7 +169,10 @@ pub fn convert_idl_type_def_to_ts(
                 let packed = modifier.packed.then(|| quote!(packed)).unwrap_or_default();
                 let align = modifier
                     .align
-                    .map(|align| quote!(align(#align)))
+                    .map(|align| {
+                        let align_literal = &Literal::usize_unsuffixed(align);
+                        quote!(align(#align_literal))
+                    })
```

Comment:
Nit to avoid indentation with new block:

```suggestion
                    .map(Literal::usize_unsuffixed)
                    .map(|align| quote!(align(#align)))
```